### DISCUSSION TITLE
Zendure general Question: different Output on Y-Adapter?

### DISCUSSION THREAD

**Discussion by busaku** [reinhard-brandstaedter/solarflow-control#230] [General]
Did anybody else notice something like that:

<img width="444" alt="image" src="https://github.com/reinhard-brandstaedter/solarflow-control/assets/25269085/dd4e32cc-2c7e-4877-93bf-e3a095d905c0">

Its nothing about solarflow-control, happens without sc too.
I use a y-Adapter and connect the hub to two input-MPPTs to the inverter

**Comment by reinhard-brandstaedter** [comment]
Yes, likely MPPTs are fighting eachother you will notice jumps in current/voltage on individual channels.

**Reply by busaku** [reply]
![image](https://github.com/reinhard-brandstaedter/solarflow-control/assets/25269085/ff57d0a2-d542-4e18-98ee-d8ce8e2e0ff6)


I had channel 3 connected via extension cable, channel 4 directly. I switched it this morning, and lo and behold: now the input is balanced.
